### PR TITLE
chore: delete owlbot.py

### PR DESCRIPTION
**Pull request by parthea**
Now that googleapis/synthtool#1244 is merged, owlbot.py is no longer required in this repo. We can add owlbot.py back in the future if repository specific customizations are needed.

### CODE REVIEW

File: owlbot.py [hunk 1]
```diff
@@ -1,110 +0,0 @@
...
-
-python.py_samples()
-
-# ----------------------------------------------------------------------------
-# Final cleanup
-# ----------------------------------------------------------------------------
-
-s.shell.run(["nox", "-s", "blacken"], hide_output=False)
-for noxfile in REPO_ROOT.glob("samples/**/noxfile.py"):
-    s.shell.run(["nox", "-s", "blacken"], cwd=noxfile.parent, hide_output=False)
```
Comment (tswast) [inline owlbot.py:110]: I'd like to keep the cleanup of sample code. I don't think the shared templates do this yet, right?

**Review by tswast — CHANGES_REQUESTED** [review]

**Comment by parthea** [comment]
Wait for https://github.com/googleapis/synthtool/pull/1276

File: owlbot.py (hunk 1 above)
Comment (parthea) [reply owlbot.py:110]: This is now part of the shared template for owlbot_main(). See googleapis/synthtool#1276

**Comment by parthea** [comment]
googleapis/synthtool#1276 was merged.

**Review by tswast — APPROVED** [review]

**Pull request merged**